rye/tests/common/mod.rs

```diff
@@ -17,7 +17,10 @@ pub const INSTA_FILTERS: &[(&str, &str)] = &[
         "[TEMP_PATH]/",
     ),
     // home
-    (r"(\b[A-Z]:)?[\\/].*?[\\/]home", "[RYE_HOME]"),
+    (
+        r"(\b[A-Z]:)?[\\/].*?[\\/]rye-test-dir[\\/]home",
```

Comment:
I need to refresh on regex syntax, but IIRC `[\\/].*?[\\/]` is redundant.

Reply:
I just noticed `[\\/].*?[\\/]` is also used similarly for the temp path filter. The `?` in `.*?` is what I was going to look at.